src/storage/heap_file.c

```diff
@@ -20434,6 +20434,20 @@ heap_get_insert_location_with_lock (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONT
       if (lk_result == LK_GRANTED)
 	{
 	  /* successfully locked! */
+#if defined (SERVER_MODE)
+	  if (lock == SCH_M_LOCK && OID_IS_ROOTOID (&context->class_oid))
+	    {
+	      /* if lock == SCH_M_LOCK, then it is the class creation */
+	      /* Schema modification lock is acquired to create Class or VClass */
+	      assert (!OID_ISNULL (&context->res_oid));
```

Comment:
Nit:

I understand your intention however, I think it can remove when I read the previous few lines.

Reply:
OK, I remove it